server/sonar-web/src/main/js/apps/issues/sidebar/SeverityFacet.js

```diff
@@ -62,12 +64,12 @@ export default class SeverityFacet extends React.PureComponent {
     this.props.onChange({ [this.property]: [] });
   };
 
-  getStat(severity: string): ?number {
+  getStat(severity /*: string */) /*: ?number */ {
     const { stats } = this.props;
     return stats ? stats[severity] : null;
   }
 
-  renderItem = (severity: string) => {
+  renderItem = (severity /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)